### PR TITLE
improvement(api): improvement logs to api 

### PR DESCRIPTION
**Pull request by danyecls**
To facilitate debugging and system analysis,
logging was implemented in error cases at the api and pkg shellhub

### CODE REVIEW

**Review by heiytor — COMMENTED** [review]
In the code, we import `logrus` as `log`. However, in this commit, different names like `logrus` and `logger` are used in different files. Maybe it would be a good idea to standardize this?

**Review by heiytor — CHANGES_REQUESTED** [review]
import `logrus` as `log`

**Review by heiytor — DISMISSED** [review]

File: api/pkg/dbtest/dbserver.go [hunk 1]
```diff
@@ -122,33 +124,37 @@ func (dbs *DBServer) start() {
...
 		if err := cmd.Run(); err != nil {
+			log.WithError(err).Warning("Failed to list running mongo containers")
+
 			return err
 		}
 
-		fmt.Fprintf(os.Stderr, "----------------------------------------\n")
-
-		panic("mongod container died unexpectedly")
+		log.Info("----------------------------------------")
```
Comment (gustavosbarreto) [inline api/pkg/dbtest/dbserver.go:156]: `log.Error`

File: api/pkg/dbtest/dbserver.go [hunk 2]
```diff
@@ -122,33 +124,37 @@ func (dbs *DBServer) start() {
...
 	if _, err := dbs.server.Process.Wait(); err != nil {
+		log.WithError(err).Warning("mongod container process wait error")
+
 		return err
 	}
 
 	if dbs.tomb.Alive() {
 		// Present some debugging information.
-		fmt.Fprintf(os.Stderr, "---- mongod container died unexpectedly:\n")
+		log.Warning("---- mongod container died unexpectedly ----")
```
[the remark this answers is not in the document]
Comment (danyecls) [reply api/pkg/dbtest/dbserver.go]: changed to log.Error

File: api/pkg/dbtest/dbserver.go [hunk 3]
```diff
@@ -122,33 +124,37 @@ func (dbs *DBServer) start() {
...
 		fmt.Fprintf(os.Stderr, "%s", dbs.output.Bytes())
-		fmt.Fprintf(os.Stderr, "---- mongod containers running right now:\n")
+		log.Info("---- mongod containers running right now ----")
```
[the remark this answers is not in the document]
Comment (danyecls) [reply api/pkg/dbtest/dbserver.go]: changed to log.Error

File: api/pkg/dbtest/dbserver.go (hunk 1 above)
Comment (danyecls) [reply api/pkg/dbtest/dbserver.go]: changed to log.Error

File: api/workers/cleaner.go [hunk 4]
```diff
@@ -22,15 +21,15 @@ import (
 func StartCleaner(_ context.Context, store store.Store) error {
 	envs, err := getEnvs()
 	if err != nil {
-		return fmt.Errorf("failed to get the envs: %w", err)
+		return err
```
Comment (henrybarreto) [inline api/workers/cleaner.go:24]: I believe we could maintain the errors built, what provider a better context on what's happened

File: api/workers/cleaner.go [hunk 5]
```diff
@@ -22,15 +21,15 @@ import (
...
 	}
 
 	if envs.SessionRecordCleanupRetention == 0 {
 		return nil
 	}
 
 	if envs.SessionRecordCleanupRetention < 0 {
-		return fmt.Errorf("invalid time interval: %w", fmt.Errorf("%d is not a valid time interval", envs.SessionRecordCleanupRetention))
+		return err
```
Comment (henrybarreto) [inline api/workers/cleaner.go:32]: In this case, the `err` does not exist, as nil, or the value does not correspond to the current problem.

I think we can maintain the `fmt.Errorf`.

File: api/workers/heartbeat.go [hunk 6]
```diff
@@ -18,12 +18,12 @@ import (
 func StartHeartBeat(_ context.Context, store store.Store) error {
 	envs, err := getEnvs()
 	if err != nil {
-		return fmt.Errorf("failed to get the envs: %w", err)
+		return err
```
Comment (henrybarreto) [inline api/workers/heartbeat.go:21]: Same here

**Review by henrybarreto — CHANGES_REQUESTED** [review]

File: api/workers/cleaner.go (hunk 4 above)
Comment (danyecls) [reply api/workers/cleaner.go]: fixed to return with fmt.

**Review by henrybarreto — APPROVED** [review]

**Review by heiytor — APPROVED** [review]

**Pull request merged**